### PR TITLE
Support comma in PhpDateTimeParser

### PR DESCRIPTION
**Pull request by thiemowmde**
This is split from https://gerrit.wikimedia.org/r/#/c/203018.

[Bug: T95532](https://phabricator.wikimedia.org/T95532)

### CODE REVIEW

File: Time.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	return 1;
 }
 
-define( 'DATAVALUES_TIME_VERSION', '0.8.0 alpha' );
+define( 'DATAVALUES_TIME_VERSION', '0.7.1 alpha' );
```
Comment (brightbyte) [inline Time.php]: This doesn't look right

Comment (thiemowmde) [inline Time.php]: That's on purpose. This is an added feature, not a breaking change, and can go in a minor release. We can increase this later if it turns out all changes should go into 0.8.

Comment (addshore) [inline Time.php]: -290148

In which cause this should be a pull request from the head revision of the 0.7.0 tag.
Right now this is pulling on top of a bunch of other changes on master https://github.com/DataValues/Time/compare/0.7.0...master
at least one of which is specified in the Changelog as changing behaviour.

This patch should a) be pulled into master without the version changed
AND
b) pulled onto a branch from 0.7.0 with a change of version to 0.7.1 and then tagged

**Comment by addshore** [comment]
-290148 in current state (see inline comment)

**Comment by thiemowmde** [comment]
I don't care much if everything goes into an 0.8.0, or non-breaking changes get release earlier as an 0.7.1. All I want is a release within the next days.

**Comment by addshore** [comment]
Well depending on what you want to do, I have continued maintaining https://github.com/DataValues/Time/tree/REL_0.7.1 with the non breaking changes

**Pull request merged**